monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -72,6 +72,13 @@ def get_agent_signals(self, agent_id: str) -> AgentSignals:
 
         return AgentSignals(**response.json())
 
+    @handle_response_parsing_errors
+    def get_agent_configuration_schema(self) -> Dict[str, Any]:
+        response = self.http_client.get("agent-configuration-schema", timeout=SHORT_REQUEST_TIMEOUT)
+        schema = response.json()
+
+        return schema
```

Comment:
It's probably worth a simple unit test that verifies `expected_schema == returned_schema`